HIRS_ProvisionerTPM2/src/CommandTpm2.cpp

```diff
@@ -150,7 +150,8 @@ void CommandTpm2::setAuthData() {
     }
 
     LOGGER.info("Attempting to set auth data.");
-    runTpm2CommandWithRetry(kTpm2ToolsTakeOwnershipCommand, argsStream.str());
+    runTpm2CommandWithRetry(kTpm2ToolsTakeOwnershipCommand, argsStream.str(),
+                            __LINE__);
```

Comment:
Seems dirty that C++ makes us do this in this way, but it is a `private` method, so that's fine I suppose.